autogpt_platform/backend/backend/data/execution.py

```diff
@@ -572,9 +573,15 @@ async def update_graph_execution_stats(
     status: ExecutionStatus,
     stats: GraphExecutionStats | None = None,
 ) -> GraphExecution | None:
-    data = stats.model_dump() if stats else {}
-    if isinstance(data.get("error"), Exception):
-        data["error"] = str(data["error"])
+    update_data: AgentGraphExecutionUpdateManyMutationInput = {
+        "executionStatus": status
+    }
+
+    if stats:
+        _stats = stats.model_dump()
+        if isinstance(_stats.get("error"), Exception):
+            _stats["error"] = str(_stats["error"])
+        update_data["stats"] = Json(_stats)
```

Comment:
nit, `_` prefix has special meaning and doesn't really fit here, so if not `data` like before maybe:
```suggestion
        stats_dict = stats.model_dump()
        if isinstance(stats_dict.get("error"), Exception):
            stats_dict["error"] = str(stats_dict["error"])
        update_data["stats"] = Json(stats_dict)
```

Reply:
what special meaning does it have?